portal/package.json

```diff
@@ -3,6 +3,7 @@
 		"@mysten/bcs": "^1.0.2",
 		"@mysten/sui": "^1.1.2",
 		"base-x": "^4.0.0",
+		"parse-domain": "^8.0.2",
```

Comment:
> we tend to only import dependencies that (i) we know are stable and well maintained, and (ii) do not offer functionality that we could quickly code and maintain ourselves

I think that it covers our use case without introducing too much complexity, so no redundancy is introduced. 

Apart from the amount of stars, I also looked at the weekly downloads (~150K), which looks good to me. 
Plus it is battle-tested from other users so it would also be likely that we have a bug in our implementation. 
At its' core, it identifies parts of the URL using the https://publicsuffix.org/ and extracts parts of the code. 

To me it looks fine as is, but would you prefer an implementation from scratch?

Reply:
Moved this one to issue https://github.com/MystenLabs/walrus-sites/issues/124 after our call.

Reply:
Closing after the discussion on this that we had offline. The package looks good, and we are tracking #124